lib/commands/clone.js

```diff
@@ -245,7 +245,7 @@ module.exports = function *(argv) {
     })
 
     yield asyncEach(tablesToCopyList, function *(table, idx) {
-      let cursor = yield sr.db(sourceDB).table(table).run({cursor: true})
+      let cursor = yield sr.db(sourceDB).table(table).run({cursor: true, timeFormat: 'raw'})
```

Comment:
Sorry, but I don't get it. Without this all records fetched would be native `Date` objects and lose timezone offset information.